paintBrush.js

```diff
@@ -0,0 +1,42 @@
+// Select the canvas in the dom
+const canvas = document.querySelector('canvas')
+// Sets the canvas to 2D drawing
+const ctx = canvas.getContext('2d')
+
+let painting = false;
+
+if (canvas) {
+    canvas.addEventListener('mousemove', drawLine);
+    // Start painting on click event
+    canvas.addEventListener('mousedown', startPainting);
+    // Stop painting on stopping clicking/off page
+    canvas.addEventListener('mouseup', stopPainting);
+    canvas.addEventListener('mouseleave', stopPainting);
+}
+
+function drawLine(event){
```

Comment:
for consistency we need a space before the `{` so it matches your other functions (and best practice)